src/extension.ts

```diff
@@ -174,15 +205,28 @@ function fetchAndLaunchMetals(context: ExtensionContext, javaHome: string) {
         javaPath,
         classpath,
         serverProperties,
-        javaOptions
+        javaOptions,
+        customRepositories
       );
     },
-    () => {
+    (reason: any) => {
```

Comment:
`any` effectively turns off type-checking, so I would prefer never using this.

Since `trackDownloadProgress` returns a `ChildProcessPromise` we can assume this has type `ExitReason` (from `promisify-child-process`)

```suggestion
    (reason: ExitReason) => {
```

<rant>I really don't like Promises because the error type is not preserved</rant>